### PR TITLE
Kudos suggestions

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Refers/Fixes

Refs: #2735

### CODE REVIEW

File: app/assets/v2/css/scroll-carousel.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#kudos-holder {
+  max-width: 498px;
+}
+
+.scroll-carousel {
+  display: flex;
+  /* padding: 3rem 0rem; */
+  /* height: 150px; */
```
Comment (thelostone-mc) [inline app/assets/v2/css/scroll-carousel.css]: can we remove this ?

File: app/assets/v2/css/scroll-carousel.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  overflow-x: scroll;
+  scroll-snap-type: x mandatory;
```
Comment (thelostone-mc) [inline app/assets/v2/css/scroll-carousel.css]: I did not know about this :D
thanks @octavioamu \m/

File: app/assets/v2/css/scroll-carousel.css (hunk 1 above)
Comment (octavioamu) [reply app/assets/v2/css/scroll-carousel.css]: 😅

**Comment by octavioamu** [comment]
Last changes
- shuffle kudos 
- fix condition when array was 1 
- removed new tag
- limited max requests until 15 kudos in the array at least

**Pull request merged**